### PR TITLE
FIX: Re-order wallet bug #3341

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4q66qpvrr056dazbuj5rgbwbz8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7203)

File: screen/wallets/list.js [hunk 1]
```diff
@@ -202,12 +203,14 @@ const WalletsList = () => {
   };
 
   const onSnapToItem = e => {
-    const contentOffset = e.nativeEvent.contentOffset;
-    const index = Math.ceil(contentOffset.x / width);
-    console.log('onSnapToItem', index);
-    if (wallets[index] && (wallets[index].timeToRefreshBalance() || wallets[index].timeToRefreshTransaction())) {
-      console.log(wallets[index].getLabel(), 'thinks its time to refresh either balance or transactions. refetching both');
-      refreshAllWalletTransactions(index, false).finally(() => setIsLoading(false));
+    if (isFocused) {
```
Comment (Overtorment) [inline screen/wallets/list.js:206]: if you invert the condition you will be able to return early, avoid many changed lines and added indentation (harder to read code)

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: screen/wallets/list.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/list.js]: Done.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/0xpkz18fa77wehbq2zyp6pu778?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7216)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**